### ISSUE TITLE
Cut r0.1.1 of s2s spec

### ISSUE THREAD

**Issue by turt2live** [matrix-org/matrix-doc#1848]
Blocked by:
* [x] https://github.com/matrix-org/matrix-doc/pull/1844 (one of the actual change that is in question)
* [x] https://github.com/matrix-org/matrix-doc/pull/1855 (another change in question)

Not blocked but nice to have:
* [x] https://github.com/matrix-org/matrix-doc/pull/1847
* [ ] https://github.com/matrix-org/matrix-doc/pull/1846
* [x] https://github.com/matrix-org/matrix-doc/pull/1845
